examples/MagicWeather/package.json

```diff
@@ -7,7 +7,8 @@
     "ios": "react-native run-ios",
     "start": "react-native start",
     "test": "jest",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "pods": "bundle install && pod-install --quiet"
```

Comment:
Maybe add `--repo-update` since a lot of the time that's required to find the latest version?